cedar-policy/src/api.rs

```diff
@@ -1248,14 +1258,15 @@ impl SchemaFragment {
         ))
     }
 
-    /// Parse a [`SchemaFragment`] from a string containing the natural schema syntax
-    pub fn from_str_natural(
+    /// Parse a [`SchemaFragment`] from a string containing the Cedar schema syntax
+    pub fn from_cedarschema_str(
         src: &str,
-    ) -> Result<(Self, impl Iterator<Item = SchemaWarning>), HumanSchemaError> {
-        let (lossless, warnings) = cedar_policy_validator::json_schema::Fragment::from_str_natural(
-            src,
-            Extensions::all_available(),
-        )?;
+    ) -> Result<(Self, impl Iterator<Item = SchemaWarning>), CedarSchemaError> {
+        let (lossless, warnings) =
+            cedar_policy_validator::json_schema::Fragment::from_cedarschema_str(
+                src,
+                Extensions::all_available(),
```

Comment:
Was about to comment here that the lossless schema should be the input `src`, but then realized that this is just an issue with the implementation, so I opened an issue instead #1118